src/components/BetaInfoBox.jsx

```diff
@@ -0,0 +1,44 @@
+/* global _ */
+import React, { useState } from 'react';
+import Alert from './ui/Alert';
+import { Button } from './ui';
+import { getBetaPopupClosed, setBetaPopupClosed } from 'src/adapters/store';
+import { parseQueryString } from '../libs/url_utils';
+
+const initialQueryParams = parseQueryString(window.location.search);
+
+const BetaInfoBox = () => {
```

Comment:
Cool! This is easier to maintain with this form, especially as a functional component.
Thanks for making this change :+1: